otx/api/entities/label_schema.py

```diff
@@ -558,6 +558,68 @@ def from_labels(cls, labels: Sequence[LabelEntity]) -> "LabelSchemaEntity":
         label_group = LabelGroup(name="from_label_list", labels=labels)
         return LabelSchemaEntity(label_groups=[label_group])
 
+    def resolve_labels_greedy(self, scored_labels: List[ScoredLabel]) -> List[ScoredLabel]:
+        """Resolves hierarchical labels and exclusivity based on a list of ScoredLabels (labels with probability).
+
+        The following two steps are taken:
+
+        - selects the most likely label from each label group
```

Comment:
```suggestion
        - select the most likely label from each label group
```
After that you have
> \- add ...

without `s`